### DISCUSSION TITLE
What should I do after receive recovery link when create new identity ?

### DISCUSSION THREAD

**Discussion by tnm0113** [ory/kratos#1003] [Q&A] [answered]
Hi all,
We are implementing an admin function with ability create normal user. I have read Manage Users and Identities on docs (https://www.ory.sh/kratos/docs/admin/managing-users-identities). but i dont know what to do next with the recovery link i receive.
How can I get csrf_token or something to submit form recovery ? I have tried curl with that link but nothing works. 
Thanks,

**Comment by k9ert** [comment, marked as answer]
Have you tried the [quickstart-tutorial](https://www.ory.sh/kratos/docs/quickstart/)? The recovery-link is working there. The link is going to the secure-app which will initiate the [recovery-flow](https://www.ory.sh/kratos/docs/self-service/flows/account-recovery).

Do you somehow want to misuse the recovery-flow to set a password for the user? I'm not sure i understand the relation to the recovery-flow.
